#Bias Removed
float64[3] gyro_bias_rem
float64[3] accel_bias_rem
float64[3] ins_bias_rem

#Gravity Corrected
float64[3] accel_g_corr

#Biases
float64[3] gyro_bias
float64[3] accel_bias
float64[3] ins_bias